### DISCUSSION TITLE
The site has JS console errors

### DISCUSSION THREAD

**Discussion by micheledurante** [searx/searx#2698] [Public Searx instances: report a bug]
**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**

**How did you install Searx?**

**What happened?**
The home throws a JS error upon first visit.

**How To Reproduce**
- Visit https://zooom.io
- Perform a search

**Expected behavior**
Any page should load without errors.

**Screenshots & Logs**
![image](https://user-images.githubusercontent.com/17530463/112768124-b5e27a00-901a-11eb-8f54-d0902b1d34cc.png)
![image](https://user-images.githubusercontent.com/17530463/112768166-e7f3dc00-901a-11eb-8fb1-1d4983500ce1.png)

**Additional context**
- Seen on Firefox version 87.0 (64-bit)
- Seen on Firefox Developer Edition version 88.0b3 (64-bit)
- At first glance, the functionality seems unaffected

**Comment by unixfox** [comment]
Again it's a Searx instance issue. When running the official Searx code there is no such error like this.
